src/operations/course.ts

```diff
@@ -0,0 +1,60 @@
+import { get } from 'http';
+import getDb from '../initDB';
+import { Course, CourseCreate, CourseUpdate,} from '../models';
+
+const db = getDb();
+
+export const createCourse = async (course: CourseCreate): Promise<Course> => {
+  return db.course.create({ data: course });
+};
+
+export const getCourseById = async (
+  id: number,
+  ownerId: string
+): Promise<Course> => {
+  try {
+    const course = await db.course.findFirst({ where: { id, ownerId } });
+    if (!course) {
+      throw new Error('Course not found');
+    }
+    return course;
+  } catch (e) {
+    throw new Error('Course not found');
+  }
+};
+
+export const getCoursesByOwnerId = async (
+  ownerId: string
+): Promise<Course[]> => {
+  try {
+    const courses = await db.course.findMany({ where: { ownerId } });
+    if (!courses) {
+      throw new Error('Courses not found');
+    }
```

Comment:
Is this check needed? My understanding of the docs is that findMany will always return a list, but this list will be empty if nothing is found.